### DISCUSSION TITLE
How to prevent app from being killed when debugging tethered iOS

### DISCUSSION THREAD

**Discussion by JGKle** [AvaloniaUI/Avalonia#15830] [Q&A]
I've got my Avalonia app up & running on Windows, Android, & iOS.

In all 3 cases, I can set breakpoints and get into the debugger (I'm developing on Visual Studio for Windows, so for Android it's a directly tethered device; for iOS it's a device that's tethered to a remote Mac).

The issue is whenever I pause the debugger on iOS, I only have like 5 seconds, and then it kills the process (with Visual Studio printing "The app has been terminated"). This makes it impossible to debug. Is there a way to have it not kill the app as soon as it pauses in the debugger?

**Comment by maxkatz6** [comment]
Possibly a question to https://github.com/xamarin/xamarin-macios/

**Reply by JGKle** [reply]
Thanks, re-posted there: https://github.com/xamarin/xamarin-macios/issues/20716
